packages/tools/examples/segmentBidirectionalTool/findLargestBidirectional.ts

```diff
@@ -0,0 +1,218 @@
+import { cache } from '@cornerstonejs/core';
+import { adaptersRT } from '@cornerstonejs/adapters';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+import vtkImageMarchingSquares from '@kitware/vtk.js/Filters/General/ImageMarchingSquares';
+import vtkDataArray from '@kitware/vtk.js/Common/Core/DataArray';
+import vtkImageData from '@kitware/vtk.js/Common/DataModel/ImageData';
+import { vec3 } from 'gl-matrix';
+
+const vtkUtils = {
+  vtkImageMarchingSquares,
+  vtkDataArray,
+  vtkImageData,
+};
+
+const EPSILON = 1e-2;
+
+export default function contourAndFindLargestBidirectional(segmentations) {
+  const { generateContourSetsFromLabelmap } = adaptersRT.Cornerstone3D.RTSS;
+
+  console.time('contour');
```

Comment:
please remove

Reply:
I think we should leave this in for a while - the timing on this can be fairly variable, and until we are reasonably confident in the implementation, I'd like to leave it.